data/wanderers.txt

```diff
@@ -158,7 +158,7 @@ outfitter "Wanderer Advanced"
 	"Sunbeam"
 	"Thunderhead Missile"
 	"Thunderhead Launcher"
-	"Thunderhead"
+	"Thunderhead Storage Array"
```

Comment:
Yeah, when I saw it I figured that MZ just put everything after `outfit` in the wanderers outfit file into the sales, because "Thunderhead" is a submunition that doesn't have a category and therefore doesn't appear in the outfitter.